packages/catalog-server/components/catalogDetail/utils.ts

```diff
@@ -0,0 +1,38 @@
+export { default as FilterOutlined } from '@ant-design/icons/FilterOutlined';
+export { default as DownOutlined } from '@ant-design/icons/DownOutlined';
```

Comment:
It was imported by the `icons.tsx` before, but I separated them because I think it will be better to have a clean and sharable component. Although we maybe no need the sharable components now 🤣